src/Core/Scheduler/Providers/PointCloudProvider.js

```diff
@@ -135,6 +135,18 @@ export default {
         if (!layer.file) {
             layer.file = 'cloud.js';
         }
+        if (!layer.group) {
+            layer.group = new THREE.Group();
+            layer.object3d.add(layer.group);
```

Comment:
you add a new group to apply transform on it?
because it doesn't work on `layer.object3d`?
if so, it should work on  `layer.object3d`

Reply:
No. Tranformations works fine on layer.object3d, and that's what user should transform. These 2 groups (layer.group and layer.bboxes) are just implementation details of  PointCloudProvider and Processing. I just moved this piece of code around, and fixed it (they were added previously to scene, which is wrong now that we have the object3d mechanism).  For reference, they are useful to hide bboxes and pointclouds independently, for example.

Reply:
It's better to tie the obbs as we do for the tiles.
There is in `tileDebug.js` an example of visibility management. it'll be nice to follow this line.
Moreover this approach will not work in the example `LAYERS TRANSFORMATIONS`

Reply:
Please note that this PR is merely a bugfix PR over existing code and structures. You want a refacto, here, right?